scripts/update-codegen.sh

```diff
@@ -18,6 +18,21 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+GOPATH_ORIG="${GOPATH}"
+SCRIPT_ROOT=$(cd `dirname "${BASH_SOURCE[0]}"`/.. && pwd)
+GOPATH_NEW="${SCRIPT_ROOT}/_go_tmp"
+GOPATH="${GOPATH_NEW}"
+TMP_DEST="${1:-}"
+mkdir ${GOPATH_NEW}
+
+cleanup() {
+    if [[ -n "${GOPATH_NEW}" && -d "${GOPATH_NEW}" ]]; then
+        chmod -R 755 ${GOPATH_NEW}
+        rm -rf "${GOPATH_NEW}"
```

Comment:
Does this mean that the code generator will be downloaded every time you run the script? Is that what we want?